src/main/java/ch/njol/skript/expressions/ExprVillagerLevel.java

```diff
@@ -0,0 +1,113 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Villager;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Villager Level")
+@Description("Represents the current level of this villager's profession. Influences the trading options generated by the villager. " +
+	"If it is greater than their profession's maximum level, no new offers are generated. Increments when the villager fills his trading xp bar. " +
+	"Also used for badge rendering. A level must be between 1 and 5, Skript will adjust when too high/low. Villagers with a level of 2 or greater " +
+	"will keep their profession regardless if they have a work site or not. If this level is 1 and the villager has no worksite, they may loose their profession.")
```

Comment:
small spelling error -> "loose" instead of "lose"

must be the Canadian spelling ;)

Reply:
You know me too well ;) 
Fixed, will be in next commit!